index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>Ahmed Reda</nav>
+      <nav><img src="./Enabled.png" alt="menu" /></nav>
```

Comment:
- [ ] I think you could wrap both of these elements in just one `nav` tag. Replace the `nav` tag of your name with an `a` tag instead that references your homepage `href="/"`. Doing it this way the semantics of your page will be improved and the user experience as well! 🚀